.github/workflows/deployment.yml

```
@@ -64,7 +64,7 @@ jobs:
           IFS=$'\n\t'
           if [[ "${{ github.repository }}" == "DIRACGrid/DIRAC" ]]; then
             if [[ "${{ github.event_name }}" == "workflow_dispatch" ]]; then
-              if [[ "${{ github.event.ref }}" =~ ^refs/heads/(integration|rel-v([8-9]|[1-9][0-9]+)\.[0-9]+)$ ]]; then
+              if [[ "${{ github.event.ref }}" =~ ^refs/heads/(integration|rel-v([8-9]|[1-9][0-9]+)p[0-9]+)$ ]]; then
```

Shouldn't this be

```suggestion
              if [[ "${{ github.event.ref }}" =~ ^refs/heads/(integration|rel-v([8-9]|[1-9][0-9]+)r[0-9]+)$ ]]; then
```
?

---

Yup 🤦